visual-tests/indicator/forceIndex.js

```diff
@@ -0,0 +1,41 @@
+(function(d3, fc) {
+    'use strict';
+
+    var data = fc.data.random.financial().startDate(new Date(2014, 1, 1))(50);
+
+    var width = 600, height = 250;
+
+    var container = d3.select('#force')
+        .append('svg')
+        .attr('width', width)
+        .attr('height', height);
+
+    // Create scale for x axis
+    var dateScale = fc.scale.dateTime()
+        .domain(fc.util.extent(data, 'date'))
+        .range([0, width])
+        .nice();
+
+    var forceAlgorithm = fc.indicator.algorithm.forceIndex();
+    forceAlgorithm(data);
+
+    // Calculate the maximum absolute value of the index
+    var aboluteMax = Math.max(Math.abs(d3.min(data, function(d) { return d.force; })),
```

Comment:
That's a good point - no, I don't see the value in duplicating what is effectively the same code within the visual tests and the website.

As a result, this would force us to do a visual inspection of the website more often (something I don't do terribly often).

More recently I have been trying to add value with the visual tests, look at the cartesian test as an example, it allows you to change the component properties and gives (or at least allows for) a more extensive test.